client/extensions/woocommerce/app/settings/shipping/save-button.js

```diff
@@ -94,7 +94,7 @@ class ShippingSettingsSaveButton extends Component {
 				</Button>
 			) : null;
 		}
-		const label = wcsEnabled ? translate( 'Save and finish' ) : translate( "I'm Finished" );
+		const label = wcsEnabled ? translate( 'Save & finish' ) : translate( "I'm Finished" );
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Finished!' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/store-styles)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).